tests/Application/Actions/Hooks/StripeChargeUpdateTest.php

```diff
@@ -367,6 +372,70 @@ public function testSuccessfulFullRefund(): void
         $webhookSecret = $container->get('settings')['stripe']['accountWebhookSecret'];
         $time = (string) time();
 
+        $chatterProphecy = $this->prophesize(StripeChatterInterface::class);
+        // Double-check that the normal success case isn't messaging Slack.
+        $chatterProphecy->send(Argument::type(ChatMessage::class))->shouldNotBeCalled();
```

Comment:
Can make a slightly stronger / more general assertion.
```suggestion
        $chatterProphecy->send(Argument::cetera())->shouldNotBeCalled();
```
I.e. we don't care what the arguments are or aren't, this method should just not be called at all.